driver-core/src/main/com/mongodb/internal/operation/FindOperation.java

```diff
@@ -874,7 +871,7 @@ private BsonDocument getCommand(final SessionContext sessionContext, final Conne
         if (showRecordId) {
             commandDocument.put("showRecordId", BsonBoolean.TRUE);
         }
-        if (allowDiskUse && (description == null || serverIsAtLeastVersionThreeDotTwo(description))) {
+        if (allowDiskUse && (description != null && serverIsAtLeastVersionThreeDotTwo(description))) {
```

Comment:
I don't think this check is necessary, since the find command itself is only sent as of version 3.2

Reply:
Ack